internal/rootcoord/meta_table.go

```diff
@@ -151,6 +151,21 @@ func (mt *MetaTable) reloadFromKV() error {
 		mt.proxyID2Meta[proxyMeta.ID] = proxyMeta
 	}
 
+	_, values, err = mt.snapshot.LoadWithPrefix(CollectionAliasMetaPrefix, 0)
+	if err != nil {
+		return err
+	}
+	aliases := make([]string, 0, len(values))
+	for _, value := range values {
+		aliasInfo := pb.CollectionInfo{}
+		err = proto.Unmarshal([]byte(value), &aliasInfo)
+		if err != nil {
+			return fmt.Errorf("rootcoord Unmarshal pb.AliasInfo err:%w", err)
+		}
```

Comment:
The error handling is tricky here. If some etcd meta is corrupted, `RootCoord` will not provide service until some manual interference is involved.

Reply:
So what‘s the correct behavior when some etcd meta is corrupted?

Reply:
I see in the unified catlog pr, the error becomes the warn log. Maybe we can first merge this pr and then do some fix.